extensions/dicom-pdf/.webpack/webpack.prod.js

```diff
@@ -42,6 +44,10 @@ module.exports = (env, argv) => {
       new webpack.optimize.LimitChunkCountPlugin({
         maxChunks: 1,
       }),
+      new MiniCssExtractPlugin({
```

Comment:
Fixes a build issue that suddenly appeared - not sure why we started needing this.